Code/Editor/Undo/Undo.cpp

```diff
@@ -49,7 +49,7 @@ class CSuperUndoStep
     }
     void Undo(bool bUndo) override
     {
-        for (size_t i = m_undoSteps.size() - 1; i >= 0; i--)
+        for (int i = static_cast<int>(m_undoSteps.size()) - 1; i >= 0; i--)
```

Comment:
[ptrdiff_t](https://en.cppreference.com/w/cpp/types/ptrdiff_t) is safer to use here than int since it will be 64 bits.

But I highly double the m_undoSteps array would be over `2^(32-1)` in the first place, so this is fine.

Reply:
leaving it as int as in some cases is operating over qt stuff that is int-based